### ISSUE TITLE
Update documentation with FSharpLu.Json instructions

### ISSUE THREAD

**Issue by danieljsummers** [giraffe-fsharp/Giraffe#342]
**TL;DR**: Include an FSharpLu.Json example in DOCUMENTATION.md under the "Customizing JsonSerializerSettings". I'd be willing to submit a PR for it.

**Discussion**: Recently, I set out to replace the JSON serializer with one that utilized FSharpLu.Json instead. [It uses SRTP, which doesn't play well with generics](https://stackoverflow.com/a/53788855/276707). After playing around with boxing and casting, [I came up with an implementation that worked](https://stackoverflow.com/a/54844636/276707).

I asked in the F# Slack for feedback on this solution, and Nino Floris shared how he uses the FSharpLu.Json converter with the built-in Newtonsoft.Json implementation. Armed with this knowledge, [I was able to successfully add the compact converter](https://github.com/bit-badger/myPrayerJournal/blob/master/src/api/MyPrayerJournal.Api/Program.fs#L35), and options are rendered much more nicely. That seems to be the "pit of success" for using FSharpLu.Json with Giraffe.

In my searching, I also came across [another question from someone who was also trying](https://stackoverflow.com/questions/41983955/using-microsoft-fsharplu-to-serialize-json-to-a-stream) to implement `IJsonSerializer` with FSharpLu.Json. I think that expanding the custom settings section to mention that you can even add custom converters, utilizing the example of the FSharpLu.Json compact converter, would save a good bit of time for people seeking to do this.

**Comment by dustinmoris** [comment]
Great suggestion! I'd be stoked if you would provide a PR to document the integratino of FSharpLu.Json with Giraffe! Thank you for the help!

**Comment by dustinmoris** [comment]
https://github.com/giraffe-fsharp/Giraffe/pull/345
